Sources/TuistKit/Services/GraphService.swift

```diff
@@ -67,6 +76,8 @@ final class GraphService {
             try exportDOTRepresentation(from: graph, at: filePath)
         case .png:
             try exportPNGRepresentation(from: graph, at: filePath, layoutAlgorithm: layoutAlgorithm)
+        case .json:
+            throw GraphServiceError.bug("\(format.rawValue) is not valid for visual export")
```

Comment:
Can't we define a proper case `jsonNotValidForVisualExport` instead of having a generic `bug` with specific message?

Reply:
Yeah that sounds good to me.